package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "nx run-many --target=lint",
     "build": "nx run-many --targets=build,build-storybook",
     "test": "nx affected:test --base master --parallel=3",
-    "test-all": "nx run-many --target=test --parallel=3",
+    "test-all": "nx run-many --target=test --parallel=3 --runInBand",
```

Comment:
<img width="1123" alt="image" src="https://github.com/wepublish/wepublish/assets/1131901/39f084b2-604d-4cb4-9c5a-5d58fff0671c">

Just for ci